be/CMakeLists.txt

```diff
@@ -152,9 +152,6 @@ set_target_properties(protoc PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/
 add_library(gtest STATIC IMPORTED)
 set_target_properties(gtest PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libgtest.a)
 
-add_library(gtest_main STATIC IMPORTED)
-set_target_properties(gtest_main PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libgtest_main.a)
-
```

Comment:
Why remove these code?

Reply:
This is just a demo.
The purpose of this PR is to merge BE unit test into one file, so that we don't need to generate binary for
all tests, which cost too much time.

Reply:
We can omit `main` function definition in test if we link gtest_main.